conf/application.conf

```diff
@@ -141,5 +141,11 @@ microservice {
       }
     }
   }
+  features {
+    enableAPIXIRouting = "Xi"
```

Comment:
Could these names be clearer?  "enableWebXIRouting" sounds like it's a boolean flag, but in reality its a value saying _where_ to route to.  Maybe e.g. api-xi-traffic-routing (or something better)?